share/smack/top.py

```diff
@@ -420,13 +420,21 @@ def verify_bpl(args):
     command += ["/maxStaticLoopBound:%d" % args.loop_limit]
     command += ["/recursionBound:%d" % args.unroll]
 
+  elif args.verifier == 'symbooglix':
+    command = ['symbooglix']
+    command += [args.bpl_file]
+    command += ["--file-logging=0"]
+    command += ["--entry-points=%s" % ",".join(args.entry_points)]
+    command += ["--timeout=%d" % args.time_limit]
+    command += ["--max-depth=%d" % args.unroll]
```

Comment:
@zvonimir Current there is no loop bound option. One could be implemented but it would require some thought. I'd want a per execution state loop bound and I'd like this to be implemented using support for execution traces which Symbooglix really could do with having.